README.md

```diff
@@ -2,38 +2,285 @@
 
-# Install ratify, with the verification key
-helm install ratify ratify/charts/ratify \
-        --set registryCredsSecret=regcred \
-        --set ratifyTestCert=$PUBLIC_KEY
-kubectl apply -f ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
+1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and select the latest release of `notation-azure-kv`. The `notation-azure-kv` binaries for each platform are available under the Assets section. Please download it based on your own platform. 
+2. Validate the checksum which should be the same as the one in `checksums.txt` and then install the plugin.
+   
+   For Linux Bash:
+   ```sh
+   version=0.6.0
 
-# Remotely sign with Azure Key Vault
-notation sign --key $KEY_NAME $IMAGE 
+   # validate checksum
+   cat checksums.txt | grep notation-azure-kv_${version}_linux_amd64.tar.gz | sha256sum -c
 
-# Deploy the image, with Gatekeeper, Ratify and Notary v2 validation of the signed image
-kubectl run net-monitor --image=$IMAGE -n demo
-```
+   # install the plugin
+   mkdir -p "$HOME/.config/notation/plugins/azure-kv"
+   tar zxf notation-azure-kv_${version}_linux_amd64.tar.gz -C "$HOME/.config/notation/plugins/azure-kv" notation-azure-kv
+   ```
+   For macOS Zsh:
+   ```sh
+   version=0.6.0
 
-See [documentation for details on remote signing with Azure Key Vault, validating a deployment to AKS with Notation and Ratify, using a simple setup script.](docs/nv2-bicep.md).
+   # validate checksum
+   cat checksums.txt | grep notation-azure-kv_${version}_darwin_amd64.tar.gz | shasum -a 256 -c
 
+   # install the plugin
+   mkdir -p "$HOME/Library/Application Support/notation/plugins/azure-kv"
+   tar zxf notation-azure-kv_${version}_darwin_amd64.tar.gz -C "$HOME/Library/Application Support/notation/plugins/azure-kv" notation-azure-kv
+   ```
+   For Windows Powershell:
+   ```powershell
+   $version = "0.6.0"
 
+   # validate checksum
+   (Get-FileHash .\notation-azure-kv_${version}_windows_amd64.zip).Hash
 
+   # install the plugin
+   mkdir "$env:AppData\notation\plugins\azure-kv"
+   Expand-Archive -Path notation-azure-kv_${version}_windows_amd64.zip -DestinationPath "$env:AppData\notation\plugins\azure-kv"
+   ```
+3. Try to run `notation plugin list` to show the installed plugin.
+
+## Getting started with a self-signed Azure Key Vault Certificate
+> **Note** It is suggested to get a certificate from a trusted CA since a self-signed certificate is not publicly trusted.
+1. Install Azure CLI by following the [guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+2. Login to Azure with Azure CLI, set the subscription and make sure the `GetCertificate` and `Sign` permission have been granted to your role:
+   ```sh
+   az login
+   az account set --subscription $subscriptionID
+   ```
+3. Create an Azure Key Vault and a self-signed certificate:
+   ```sh
+   resourceGroup=notationResource
+   keyVault=notationKV
```

Comment:
"The vault name 'notationKV' is already in use. Vault names are globaly unique so it is possible that the name is already taken." 
Maybe we should suggest users to choose own Vault name

Reply:
Agreed,  add a comment stating this must be a unique value and split the KV creation and cert creation into separate steps. If the user has a copy/paste fail b/c they didn't update the value, it'll be easier to troubleshoot.

Reply:
Updated.